assets/scss/1_base/4-buttons.scss

```diff
@@ -0,0 +1,27 @@
+.btn {
+  display: inline-block;
+  border: 0;
+  outline: 0;
+  box-shadow: none;
+  padding: 5px 10px;
+  cursor: pointer;
+  border-radius: 2px;
+  background: $gray;
+  transition: bacground 0.3s ease;
+  will-change: bacground;
```

Comment:
👍 Wasn't even aware of this rule. 😄   
Although, documentation suggest using it sparingly and maybe only when we face performance issues?
https://developer.mozilla.org/en-US/docs/Web/CSS/will-change
I'd reckon that such transitions as background or colour changes aren't that computationally important for browsers.
TL.DR.: Feel free to leave it in, but in future - consider where and if it is necessary. 😉

Reply:
True, is better to use for advanced transitions e.g. with transforms. Thanks.